app/views/donations/edit.html.erb

```diff
@@ -26,6 +26,12 @@
 <!-- Main content -->
 <section class="content">
   <div class="container-fluid">
+    <% if @audit_performed_on_donated_items %>
+      <div class="card-header border-0 bg-gradient-warning">
+        You’ve had an audit since this donation was started. In the case that you are correcting a typo, rather than recording that the physical amounts being (purchased/donated) have changed,
```

Comment:
```suggestion
        You’ve had an audit since this donation was started. In the case that you are correcting a typo, rather than recording that the physical amounts being donated have changed,
```